integration_tests/src/main/python/arithmetic_ops_test.py

```diff
@@ -876,8 +907,7 @@ def test_acosh(data_gen):
 @pytest.mark.parametrize('data_gen', [DoubleGen(min_exp=-20, max_exp=20)], ids=idfn)
 def test_columnar_acosh_improved(data_gen):
     assert_gpu_and_cpu_are_equal_collect(
-            lambda spark : unary_op_df(spark, data_gen).selectExpr('acosh(a)'),
-            {'spark.rapids.sql.improvedFloatOps.enabled': 'true'})
```

Comment:
Why are we removing this?  The test is explicitly trying to test when this config is true.  Removing is also inconsistent with other tests below that explicitly set this to true.

Reply:
I removed it because `spark.rapids.sql.improvedFloatOps.enabled` is now set to true by default. I can add it back if we want to make it explicit.

Reply:
Well, I agree it's better to keep it. Added this and another back in https://github.com/NVIDIA/spark-rapids/pull/11283/commits/04d86b8f596daf9a0743894cf165ace55af13007.